Commands.adoc

```diff
@@ -0,0 +1,244 @@
+`start`:: Starts the given service in the VM +
+`stop`:: Stops the given service in the VM +
+`restart`:: Restarts the given service in the VM +
+
+The following options can be used: +
+
+`--script-readable`:: Displays information in a key=value format +
+`-h, --help`:: Prints help for the specific command being run +
+
+
+== Get Environment Variables and Certificates
+
+If you need to get the environment variables and certificates for all the active
+ services in the VM, run: +
+
+`$ vagrant service-manager env` +
+
+Usage: `vagrant service-manager env [service] [options]`
+
+The possible options for service are `docker` and `openshift`. +
+If no service is specified `vagrant service-manager env` provides connection
+ information for all the active services.
+
+The following options can be used: +
+
+`--script-readable`:: Displays information in a key=value format +
+`-h, --help`:: Prints help for the specific command being run. +
+
+=== Examples
+
+If you need connection information for all active services in the VM, in a
+manner that can be evaluated in a shell, use:
+-----------------------------------------------------------------------------
+$ vagrant service-manager env
+
+# docker env:
+# Set the following environment variables to enable access to the
+# docker daemon running inside of the vagrant virtual machine:
+export DOCKER_HOST=tcp://10.1.2.2:2376
+export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+export DOCKER_TLS_VERIFY=1
+export DOCKER_API_VERSION=1.21
+
+# openshift env:
+# You can access the OpenShift console on: https://10.1.2.2:8443/console
+# To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+export OPENSHIFT_URL=https://10.1.2.2:8443
+export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+export DOCKER_REGISTRY=hub.openshift.centos7-adb.10.1.2.2.xip.io
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env)"
+-----------------------------------------------------------------------------
+
+If you need to run the plugin and get the environment variables and
+certificates for a specific service, use: +
+For Docker:
+----------------------------------------------------------------------------
+$ vagrant service-manager env docker
+# Set the following environment variables to enable access to the
+# docker daemon running inside of the vagrant virtual machine:
+export DOCKER_HOST=tcp://172.28.128.3:2376
+export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+export DOCKER_TLS_VERIFY=1
+export DOCKER_API_VERSION=1.21
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env docker)"
+----------------------------------------------------------------------------
+
+*Note:* In the case of the `docker` service, the required TLS certificates for
+securing the Docker communication will be regenerated and copied to the host
+as part of the command execution. If existing certificates are found to be
```

Comment:
@bexelbie More clearer picture is. TLS certs will generate as a part of `vagrant up` process (you will see message along with other messages).
It will be regenerated on `env docker` execution if it is invalid or doesn't exists.